go-controller/pkg/ovn/base_network_controller_policy.go

```diff
@@ -851,6 +833,15 @@ func (bnc *BaseNetworkController) getNetworkPolicyPortGroupDbIDs(namespace, name
 		})
 }
 
+func parsePGPolicyKey(pgPolicyKey string) (string, string, error) {
```

Comment:
This looks like it should be two separate IDs?

Reply:
not sure though if it will be more confusing that some objects have object namespace+name in this field, and some have them separately, wdyt?

Reply:
I updated it to use shared function

Reply:
nit: I am wondering if the new `BuildNamespaceNameKey` could be reused else where as we should be building this Id for other objects already.